### PR TITLE
:sparkles: Add back 'Change' messages in sink create

### LINKED ISSUE #606: Add back "Changes"/"Rows" selection
* When the table is an events table, auto-select rows (disallow changing)

### PR DESCRIPTION
**Pull request by acco**
Closes #606

### CODE REVIEW

**Comment by acco** [comment]
* **#630** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/630?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/630?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: assets/svelte/consumers/ShowSink.svelte [hunk 1]
```diff
@@ -220,13 +220,15 @@
...
       {/if}
 
-      <Cursor
-        {cursor_position}
-        messages_processed_count={metrics.messages_processed_count
-          ? metrics.messages_processed_count.toLocaleString()
-          : 0}
-        {onRewind}
-      />
+      {#if consumer.message_kind === "record"}
```
Comment (acco) [inline assets/svelte/consumers/ShowSink.svelte:223]: Disable cursor position for changes for now

File: assets/svelte/consumers/SinkConsumerForm.svelte [hunk 2]
```diff
@@ -138,6 +140,13 @@
       sortColumnName = selectedTable.columns.find(
         (column) => column.attnum === form.sortColumnAttnum,
       )?.name;
+
+      // Force message kind to "record" for event tables
+      if (selectedTable.is_event_table) {
+        form.messageKind = "record";
+      } else {
+        form.messageKind = "event";
```
Comment (acco) [inline assets/svelte/consumers/SinkConsumerForm.svelte:148]: When you change tables, we flip this back to `event`

File: assets/svelte/consumers/SinkConsumerForm.svelte [hunk 3]
```diff
@@ -342,6 +460,59 @@
...
+                Select the kind of messages you want to process.
+                <!-- <a
+                  href="https://sequinstream.com/docs/reference/messages"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  class="inline-flex items-center text-link hover:underline"
+                >
+                  Learn more
+                  <ExternalLinkIcon class="w-3 h-3 ml-1" />
+                </a> -->
```
Comment (acco) [inline assets/svelte/consumers/SinkConsumerForm.svelte:475]: We'll add back docs links when docs are out

File: lib/sequin_web/live/sink_consumers/show.ex [hunk 4]
```diff
@@ -379,14 +379,18 @@ defmodule SequinWeb.SinkConsumersLive.Show do
...
   def handle_info(:update_cursor_position, socket) do
-    task =
-      Task.Supervisor.async_nolink(
-        Sequin.TaskSupervisor,
-        fn -> get_cursors(socket.assigns.consumer) end,
-        timeout: :timer.seconds(30)
-      )
-
-    {:noreply, assign(socket, :cursor_task_ref, task.ref)}
+    if socket.assigns.consumer.message_kind == "record" do
```
Comment (acco) [inline lib/sequin_web/live/sink_consumers/show.ex:382]: disable fetch of cursor position for changes

Comment (RTLS) [reply lib/sequin_web/live/sink_consumers/show.ex:382]: I modified the get_cursors function to return an empty map- I will hook into this lifecycle for updating backfills in the future. so you can hopefully re-enable it for events when rebasing

**Review by RTLS — APPROVED** [review]

**Pull request merged**